README.md

```diff
@@ -25,16 +19,26 @@ saucectl version 0.4.0
 (build 7468a24c788b4ca4d67d50372c839edf03e5df6a)
 ```
 
-__Note:__ if you run the command for the first time it will initially download the binary for you. This only happens once.
+__Note:__ When you run the command for the first time, it will initially download the binary. This only happens once.
+
+__Note:__ `saucectl` installation is disabled on Sauce Labs Cloud. If you wish to force the installation, set the `FORCE_INSTALL_SAUCECTL` environment variable to `true`.
```

Comment:
Maybe `SAUCECTL_FORCE_INSTALL` instead to match the pattern of the other env vars (e.g. `SAUCECTL_INSTALL_BINARY`)